prometheus/src/bank_metrics.rs

```diff
@@ -0,0 +1,49 @@
+use solana_runtime::bank::Bank;
+
+use crate::utils::{write_metric, Metric, MetricFamily};
+use std::{io, sync::Arc, time::SystemTime};
+
+pub fn write_bank_metrics<W: io::Write>(
+    at: SystemTime,
+    bank: &Arc<Bank>,
+    out: &mut W,
+) -> io::Result<()> {
+    write_metric(
+        out,
+        &MetricFamily {
+            name: "solana_bank_slot",
+            help: "Current Slot",
+            type_: "gauge",
+            metrics: vec![Metric::new(bank.slot()).at(at)],
```

Comment:
If I understand correctly, Solana tracks multiple banks for the multiple commitment levels (the finalized one, the confirmed one, etc.)

I think it would make sense to expose metrics for all of them with different labels, e.g. `solana_bank_slot{commitment="finalized"}`, `solana_bank_slot{commitment="unconfirmed"}`, etc.

Reply:
Can we do this at a later point? I can't find the API in the structs, have to do some digging on which slots to get from banks.

Reply:
Exposing all of `{unconfirmed,confirmed,finalized}` separately, yes we can do that at a later point, one of them is fine for now.

But we do have to know which one it is currently! If we don’t know which bank we get, then we don’t know what the metric means! We should also add the label already, so that there is no ambiguity in the data once we start collecting it.